### ISSUE TITLE
🔦 Feature: Web view like milanote or miro

### ISSUE THREAD

**Issue by AztrexDX** [nocodb/nocodb#6548]
### Please confirm if feature request does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Describe the usecase for the feature

A web view like milanote or miro where we can place items in grid and connect it

### Suggested Solution

A new web view

**Comment by o1lab** [comment]
Can you clarify more on this - did you mean to say having a canvas where you drag an drop notes / objects to make diagram. How does it with fit a DB view ?

**Comment by AztrexDX** [comment]
yes. something like be able to place items on canvas like in obsidian and be able to see the items data. Like in obsidian we can see the text items and connect
![image](https://github.com/nocodb/nocodb/assets/86340924/7aa707a4-de2e-4e5d-abbd-09777470165a)

**Comment by nileshtrivedi** [comment]
This might make more sense as a per-database view (with foreign keys becoming links), instead of a per-table view.
Though, the problem with graph views is that they quickly become illegible (above 100 or so nodes) unless either nesting or hierarchical clustering is implemented.

**Comment by dstala** [comment]
Support for documentation is something that we are considering (markdown style) with option to add images. But Obsidian style is not in the roadmap. Will tag this issue to docs request & will be tracked when that's picked up.
